pytaboola/services/report.py

```diff
@@ -20,11 +20,6 @@ def __prepare_filters(self, start, end, **filters):
         assert isinstance(end, (datetime, date)), \
             "End date filter must be a date/datetime instance"
 
-        if isinstance(start, datetime):
```

Comment:
Removing this cast so that datetime parameters for hourly reporting can be passed through